lib/apollo-federation/service.rb

```diff
@@ -5,6 +5,9 @@
 module ApolloFederation
   class Service < GraphQL::Schema::Object
     graphql_name '_Service'
+    description 'The sdl representing the federated service capabilities. Includes federation ' \
```

Comment:
Should `sdl` be capitalized?

Reply:
I copied the description directly from the JS `@apollo/federation` lib: https://github.com/apollographql/apollo-server/blob/master/packages/apollo-federation/src/types.ts#L28